### DISCUSSION TITLE
Downloading packages from Bintray fails

### DISCUSSION THREAD

**Discussion by Yogiraj-Suryakant-Tambake** [rabbitmq/rabbitmq-server#3019] [Other]
Steps to reproduce

wget https://dl.bintray.com/rabbitmq/all/rabbitmq-server/3.7.14/rabbitmq-server-3.7.14-1.el7.noarch.rpm
or 
curl https://dl.bintray.com/rabbitmq/all/rabbitmq-server/3.7.14/rabbitmq-server-3.7.14-1.el7.noarch.rpm

You will receive 403 Forbidden Status.

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
Bintray has been [shut down on May 1st](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/). We have [announced where we are migrating](https://blog.rabbitmq.com/posts/2021/03/migrate-off-of-bintray/) in late March.

**Reply by Yogiraj-Suryakant-Tambake** [reply]
Thanks for the quick reply @michaelklishin .
